.github/actions/version-check/action.yml

```diff
@@ -29,9 +29,9 @@ runs:
         TRIMMED_VERSION=$(echo "$VERSION" | xargs) 
 
         if ${{ env.IS_PRERELEASE == 'true' }}; then
-          VERSION_FORMAT="^[0-9]\.[0-9]\.[0-9]-test-[1-9]$"
+          VERSION_FORMAT="^[0-9]+\.[0-9]\.[0-9]-test-[1-9]$"
```

Comment:
Assuming we won't add the test version more than 3, besides that last one, we might want to add `+` after each of the `[0-9]`, or it could break with the number over 9. For example, `10.10.10-test-1`

Reply:
Sorry, I didn't notice Jordan's comment.

Reply:
I don't think `x.10.z` would ever happen given the way we follow the versioning in Transact, but `x.y.10` can happen yes

Reply:
Fixed with 51f4640bbed0b0730850add21c0ace3a976baae4